code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -37,6 +37,13 @@
 	weaponscheck = 0
 	auto_patrol = 1
 
+/mob/living/simple_animal/bot/secbot/beepsky/explode()
+	var/turf/Tsec = get_turf(src)
+	new /obj/item/weapon/stock_parts/cell/potato(Tsec)
+	var/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/G = new(Tsec)

```

Comment:
Isn't there an actual shotglass? Would probably be better to use that than a regular drinking glass.